cmd/buildtools/openebs.go

```diff
@@ -88,3 +152,104 @@ var updateOpenEBSAddonCommand = &cli.Command{
 		return nil
 	},
 }
+
+var updateOpenEBSImagesCommand = &cli.Command{
+	Name:      "openebs",
+	Usage:     "Updates the openebs images",
+	UsageText: environmentUsageText,
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:  "openebs-version",
+			Usage: "The version of openebs to use to determine image versions",
+		},
+		&cli.StringFlag{
+			Name:  "kubectl-version",
+			Usage: "The version of kubectl to use to determine image versions",
+		},
+	},
+	Action: func(c *cli.Context) error {
+		logrus.Infof("updating openebs images")
+
+		rawver, err := GetMakefileVariable("K0S_VERSION")
+		if err != nil {
+			return fmt.Errorf("failed to get k0s version: %w", err)
+		}
+		k0sVersion := semver.MustParse(rawver)
+
+		logrus.Infof("fetching wolfi apk index")
+		wolfiAPKIndex, err := GetWolfiAPKIndex()
+		if err != nil {
+			return fmt.Errorf("failed to get APK index: %w", err)
+		}
+
+		current := openebs.Metadata
+		newmeta := release.AddonMetadata{
+			Version:  current.Version,
+			Location: current.Location,
+			Images:   make(map[string]string),
+		}
+
+		values, err := release.GetValuesWithOriginalImages("openebs")
```

Comment:
nit: correct me if i'm wrong, but from this line onwards, the logic is the same as `updateOpenEBSAddonCommand`, right?